Traceback (most recent call last):
  File "/usr/bin/certbot", line 11, in <module>
    load_entry_point('certbot==0.10.2', 'console_scripts', 'certbot')()
  File "/usr/lib/python2.7/dist-packages/certbot/main.py", line 838, in main
    _post_logging_setup(config, plugins, cli_args)
  File "/usr/lib/python2.7/dist-packages/certbot/main.py", line 812, in _post_logging_setup
    hooks.validate_hooks(config)
  File "/usr/lib/python2.7/dist-packages/certbot/hooks.py", line 19, in validate_hooks
    validate_hook(config.post_hook, "post")
  File "/usr/lib/python2.7/dist-packages/certbot/hooks.py", line 49, in validate_hook
    raise errors.HookCommandNotFound(msg)
HookCommandNotFound: Unable to find post-hook command ./letsencrypt_renewhook.sh in the PATH.
(PATH is /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin)
